tests/unit/twitter.js

```diff
@@ -4,13 +4,14 @@
   var deps = [
     'assert',
     'nock',
+    'underscore',
```

Comment:
Anyway, just replace `'underscore'` with `'lodash'` and it will be fine.

Docs are here https://lodash.com/docs#assign